src/Command/BakeCommand.php

```diff
@@ -164,4 +164,28 @@ protected function deleteEmptyFile(string $path, ConsoleIo $io): void
             $io->out(sprintf('<success>Deleted</success> `%s`', $path), 1, ConsoleIo::NORMAL);
         }
     }
+
+    /**
+     * Check if a column name is valid
+     *
+     * The Regex used here basically states that:
+     * the column name has to start with a character (lower or upper case) or an underscore and
+     * further characters are allowed to be either lower or upper case characters, numbers or underscores
+     *
+     * Anything else like
+     * - any other special character
+     * - umlauts
+     * - whatever the user thinks is a good idea to use in a table name (emojis?)
+     *
+     * is prevented with this check.
+     *
+     * @param string $name The name of the column
+     * @return bool true, if it is valid, false if not
+     */
+    protected function isValidColumnName(string $name): bool
+    {
+        $re = '/^[a-zA-Z_][a-zA-Z0-9_]*$/';
```

Comment:
The regex is short enough, so can directly use as argument instead of assigning to a variable first.

Reply:
:+1: We gotta save those op-codes 💸